packages/graphql/src/translate/create-aggregate-where-and-params.ts

```diff
@@ -253,12 +254,21 @@ function createEntityOperation(
         });
     } else {
         const innerVar = new Cypher.Variable();
-        const innerOperation = createBaseOperation({
-            operator: logicalOperator || "EQ",
-            property: innerVar,
-            param: paramName,
-        });
 
+        let innerOperation: Cypher.Operation;
+        if (fieldType === "Duration") {
+            innerOperation = createDurationOperation({
```

Comment:
I've got a feeling the mistake we made here was using `createBaseOperation` instead of `createComparisonOperation`.

That also seems to do something different for `Point` fields which I'm guessing is something else that doesn't work as it should with aggregations.

It looks like it might be a bit annoying to pass the `context.neo4jDatabaseInfo` here but I think it's worth it to keep a single source of truth for handling these operations

Reply:
I agree that this is probably the root cause of this problem, however it is not a quick and simple refactor I think, as not only the context but the format of the provided fields need to be different

I agree with the `Point` and I was planning on addressing that in a separate PR

Reply:
Oh, we need to fix it properly, but this is not going to increase our technical debt towards that so no sense in having that broken while we fix it properly.

The points on separate PR is simply to avoid large PRs, as point tests need to be added to guarantee it is a problem that need fixing